lang/LangPrimSource/PyrPrimitive.cpp

```diff
@@ -3528,6 +3528,14 @@ static int prLanguageConfig_removeExcludePath(struct VMGlobals * g, int numArgsP
 	return prLanguageConfig_removeLibraryPath(g, numArgsPushed, excludePaths);
 }
 
+static int prLanguageConfig_getCurrentConfigPath(struct VMGlobals * g, int numArgsPushed)
+{
+	PyrSlot *a = g->sp;
+	PyrString* string = newPyrString(g->gc, gLanguageConfig->getCurrentConfigPath(), 0, false);
```

Comment:
Yes, I'll put together a pull request tomorrow if I have time. I'll fix the
c++ method

I guess calling store with nil causes it to magically create one with the
default path.

On Wed, Mar 25, 2015 at 8:34 PM Julian Rohrhuber <notifications@github.com>
wrote:

> In lang/LangPrimSource/PyrPrimitive.cpp
> <https://github.com/supercollider/supercollider/pull/1327#discussion_r27157295>
> :
>
> > @@ -3528,6 +3528,14 @@ static int prLanguageConfig_removeExcludePath(struct VMGlobals * g, int numArgsP
> >  	return prLanguageConfig_removeLibraryPath(g, numArgsPushed, excludePaths);
> >  }
> >
> > +static int prLanguageConfig_getCurrentConfigPath(struct VMGlobals * g, int numArgsPushed)
> > +{
> > +	PyrSlot *a = g->sp;
> > +	PyrString* string = newPyrString(g->gc, gLanguageConfig->getCurrentConfigPath(), 0, false);
>
> Ah I assumed it would always exist, which is obviously wrong. You know how
> to fix?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/supercollider/supercollider/pull/1327/files#r27157295>
> .
>